lib/api/dsl/storage/index.js

```diff
@@ -230,16 +229,16 @@ class Storage {
    */
   remove (filterId) {
```

Comment:
nitpicking: change the jsdoc comment: this method does not return a Promise anymors